crates/subspace-rpc-primitives/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "subspace-rpc-primitives"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Please add other fields we typically have in such crates and a `LICENSE` file

Reply:
Done. Hmm, but do we really have to add a `LICENSE` file per crate? I know different crates can have different licenses, why not just put all these LICENSE files on the project root without copying them around?

Reply:
I guess we can do that, but:
* it makes number of files in the root folder (which is a scarce resource) bigger
* it is easier to maintain when everything related to the crate is contained in its directory and not somewhere else

Reply:
Just thought it's a bit of waste in terms of the repo size, especially when we're going to have more and more crates.

Reply:
I guess if we have license specified in the `Cargo.toml` we can skip the file, but AFAIK folks from Web3 Foundation didn't like that when they did grant review for instance :shrug: